GSoC-2021/Anoushka-Ramesh/README.md

```diff
@@ -0,0 +1,88 @@
+# GSoC2021 Expression of Interest: Anoushka Ramesh
+Email - anoushkaramesh21@gmail.com
+
+## Assessment Step 1
+Website - [https://gsocautspaces.herokuapp.com](https://gsocautspaces.herokuapp.com)
+
+Activity page - [https://www.openhumans.org/activity/project-autspaces/#activity-panel-data](https://www.openhumans.org/activity/project-autspaces/#activity-panel-data)
+
+UserData JSON file - 
+![Screenshot from 2021-03-29 22-59-34](https://user-images.githubusercontent.com/73518403/112876035-a0388780-90e2-11eb-8e98-fd2a71a55d11.png)
+
+## Assessment Step 2
+[Todo-App](https://github.com/anoura12/todo-app) - A basic Django todo application where you can create and update tasks.
+I've used HTML and Tailwind CSS for the frontend.
+
+## Assessment Step 3
+The development requirement that I would like to work on is the additional design requirements for the website.
+
+Most of the world’s autistic population uses the web for their personal needs, and the number keeps increasing every year. Most websites contain bright colors, glary screens, ambiguous symbols, and cluttered layouts that overwhelm autistic users because this triggers their sensory processing sensitivities. Hence, I would like to create a simple but effective user interface that makes it easier for them to navigate the Autistica website. 
+
+I have gone through Sprint No. 6 where the site was tested on autistic people/parents of autistic people and I have concluded that the website design can be broken down into 4 categories-
+1. Layout
+2. Color
+3. Text
+4. Symbols
```

Comment:
Thank you so much for digging into the documentation on the repository so well @anoura12!

Unfortunately these wireframes are quite out of date. However, I totally follow your logic on how they can be improved!

If we take your interest forward to a GSOC application I think I'd like for you to think more broadly about an accessible layout, rather than editing these older wireframes.

Does that sound ok? You can absolutely use them as jumping off points but I think we'd want to design the user interface to have a coherent style across the app.